packages/core/src/Loading/Loading.styles.tsx

```diff
@@ -19,7 +20,7 @@ export const { staticClasses, useClasses } = createClasses("HvLoading", {
     },
 
     ":has($small)": {
-      "--scaleY": "0.223",
+      "--scaleY": "60%",
```

Comment:
This should `22%`, to match what we had. Also, let's call the var `height` or `h` - it's not longer a scale
```suggestion
      "--height": "22%",
```